Dockerfile

```diff
@@ -1,9 +1,21 @@
 FROM ruby:3.0.2-alpine
-WORKDIR /code
+
+ENV APP_DIR /opt/app
+
+RUN mkdir -p $APP_DIR
+
+WORKDIR $APP_DIR
+
 RUN bundle config set --local without 'development test'
+
 RUN apk update
+
 RUN apk add --virtual build-dependencies build-base ruby-dev libpq postgresql-dev tzdata
-COPY Gemfile /code
-RUN bundle install 
-COPY . /code
-CMD rails s -b 0.0.0.0 -p ${RAILS_PORT:-80} -e production
\ No newline at end of file
+
+COPY Gemfile $APP_DIR
+
+RUN bundle install
+
+COPY . $APP_DIR
+
+CMD rails s -b 0.0.0.0 -p 3000 -e production
```

Comment:
You can specify the port forwarding in docker-compose or using docker itself, then why should we have it here? 

Besides, IMO dockerfiles should in most cases have the same output no matter what. Using RAILS_PORT var is not needed here and also against that principle.

Reply:
You should ask @wojtodzio about it because he ordered this change. It may be more professional to have it regulated by env variables insted of docker port forwarding?

Reply:
I guess @micorix is right, I was wrong to suggest adding that variable previously